### PR TITLE
The Hatchet can be used to perform cool takedowns when thrown

### PR DESCRIPTION
**Pull request by Tacolizard**
**When thrown it will do some decent damage, and knock them down. Has a 75 percent embed chance, so it will rarely be able to be thrown more than once. You'll also scream a custom warcry.**

just imagine how scary it would be to be greytiding and having some bandana wearing botanist take out another grey with a tomahawk and start whooping at you.

**i can't test locally for a while so if someone could for me that'd be great**

Balance wise, should this be a high damage weapon that knocks you down or a low damage weapon that just stuns for a second?

i am more in favor of the former because it'll look awesome.

### CODE REVIEW

**Comment by XDTM** [comment]
Botanists can finally show off their true roots

**Comment by ma44** [comment]
_Deep breathing_

**Comment by XDTM** [comment]
Status is a var to check if stunbatons are on, you shouldn't check for it in hatchets.

**Comment by Tacolizard** [comment]
i had an idea. I wanna make it so you can set a warcry like the explosive lance.

**Comment by TechnoAlchemisto** [comment]
Giving these a weaken is way too good considering how easy they are to snag

**Comment by ExcessiveUseOfCobblestone** [comment]
Sprite?

File: code/modules/hydroponics/hydroitemdefines.dm
Comment (ExcessiveUseOfCobblestone) [inline code/modules/hydroponics/hydroitemdefines.dm]: excuse you

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]

**Comment by PKPenguin321** [comment]
Thrown weakens are already too widespread as it is. :-1: Way too huge of a buff

File: code/modules/hydroponics/hydroitemdefines.dm
Comment (PKPenguin321) [inline code/modules/hydroponics/hydroitemdefines.dm]: I'm not sure you realize how fucking ridiculous this is

**Comment by ExcessiveUseOfCobblestone** [comment]
botany powercreep 😉

**Comment by Tacolizard** [comment]
@PKPenguin321 i know it's powerful. i willing to compromise to get this merged. if you would like this nerfed, please suggest an alternative.

**Comment by Tacolizard** [comment]
@PKPenguin321 according to [https://tgstation13.org/wiki/Guide_to_Combat#Defense](url) there aren't many thrown weakens. and the stunbaton only has a 35 percent chance of a thrown weaken.

**Comment by Tacolizard** [comment]
also i have lowered all the stuns and damage a bit

**Comment by Tacolizard** [comment]
kek. i just fixed it f

**Comment by TechnoAlchemisto** [comment]
They shouldn't have a stun at all, really.

**Comment by optimumtact** [comment]
This is unlikely to get merged as it is a ranged stun, and please stop closing and reopening your pr's

**Comment by Tacolizard** [comment]
@optimumtact sorry i was making a comment and accidentally clicked the close and comment button.  
also when you and @TechnoAlchemisto say no stun at all do you mean no stun or no weaken?

**Comment by optimumtact** [comment]
they acheive the same goal really

**Comment by XDTM** [comment]
Use weaken so they fall down, instead of standing stunned on foot.
I think you could raise embedded_impact_pain_multiplier a bit, it's basically removing your weapon when you embed it after all. For reference, the spear has it at 3, which multiplied by the w_class (4) deals 12 extra damage and requires no autolathe to make.

**Comment by TechnoAlchemisto** [comment]
Except spears can't be stored in your pocket or in your backpack, easily accessible pocket stuns are bad

**Comment by Tacolizard** [comment]
@XDTM i really couldn't agree with you more, this is not overpowered at all

@TechnoAlchemisto you can literally build a spear in the permabrig, they're so easy to make. And that's a weapon with the power to break down doors and destroy shit. Only two hatchets spawn on box roundstart, others would need to be autolathed. And the hatchet would still be less powerful than the spear.

**Comment by Tacolizard** [comment]
new changes:

force lowered from 12 to 10
throwforce increased from 18 to 20
embed chance lowered from 75 to 50 (still unsure where exactly this should be set)
embed pain multiplier increased from 2 to 4

**Comment by TechnoAlchemisto** [comment]
Last time I checked there weren't cable coils in perma, but whatever.

The stun is the problem, if you want to make hatchets better just up their force, it makes sense for them to be good weapons.

**Comment by Tacolizard** [comment]
@TechnoAlchemisto cultivator to break reinforced glass pane, use metal rod to break machinery such as seed harvestors, collect coil.

File: code/modules/hydroponics/hydroitemdefines.dm
Comment (PKPenguin321) [inline code/modules/hydroponics/hydroitemdefines.dm]: this absolutely needs to not weaken. this is way, way too good. thrown shit bypasses any defense including shields, dual eswords, etc and is super hard to dodge.

Comment (Iamgoofball) [inline code/modules/hydroponics/hydroitemdefines.dm]: You see how it checks shields above

Comment (PKPenguin321) [inline code/modules/hydroponics/hydroitemdefines.dm]: still its ridiculously good, this and a bola for example is basically a free kill with minimal effort in collecting weapons

Comment (Tacolizard) [inline code/modules/hydroponics/hydroitemdefines.dm]: well you loose the weapon when it embeds so you can only stun once

Comment (PKPenguin321) [inline code/modules/hydroponics/hydroitemdefines.dm]: it's free as in with other stuns they require some effort to acquire, i think one or two of these are in the garden roundstart for literally any powergamer to pocket and botanists have like 4 in the back for free

**Comment by PKPenguin321** [comment]
Very strongly opposed

File: code/modules/hydroponics/hydroitemdefines.dm
Comment (Tacolizard) [inline code/modules/hydroponics/hydroitemdefines.dm]: only one is in the garden and the botanists only have two

**Comment by drynwyn** [comment]
With PK on this. A thrown stun is way too powerful. Just make it do good damage with a high embed chance.

**Comment by AdamElTablawy** [comment]
oh boy another good reason not to buy a throwing weapon kit because hatchets will basically do the same job and aren't illegal

Hmmmm

Tbh i'd say lower damage and change any form of stun to increased bleeding

Perhaps have it do more damage on targets in red health when thrown, so that you can finish off a fleeing enemy with a thrown hatchet, but not use them as a default god tier weapon

**Comment by FantasticFwoosh** [comment]
***cough*** ashwalker navajo viva 2k16 bone tomahawks when? ***cough***

**Comment by Iamgoofball** [comment]
Yeah, focus more on the embed

**Comment by ExcessiveUseOfCobblestone** [comment]
at what point does the sexy [but frowned upon] native american costumes come out?

**Comment by Cheridan** [comment]
Don't merge this as long as it stuns

**Comment by Tacolizard** [comment]
@Cheridan what about if the the weaken is replaced with a one second stun. (you stay standing)

**Comment by Iamgoofball** [comment]
Sounds good

**Comment by PKPenguin321** [comment]
Stun is just a weaken with less visual feedback that doesn't make you immune to lasers being shot over you

**Comment by Tacolizard** [comment]
@PKPenguin321 yeah but it only lasts 1 second so it's just for dramatic effect, dosn't really do much

**Pull request closed without merging**
